scripts/boto3/multithread_download/uh_download.py

```diff
@@ -69,7 +69,10 @@ def list_objects(self, bucket):
                 yield entry
 
     def push(self, bucket, key):
-        local_path = self.config.path / bucket / key
+        if (self.config.path == pathlib.Path('.')):
+            local_path = self.config.path / bucket / key
+        else:
+            local_path = self.config.path / key
```

Comment:
When one is downloading multiple buckets, this will put all of them in one directory, overwriting files with same name.

Reply:
reverted the behaviour of the download script